internal/datastore/crdb/watch.go

```diff
@@ -28,6 +28,17 @@ func (cds *crdbDatastore) Watch(ctx context.Context, afterRevision datastore.Rev
 	updates := make(chan *datastore.RevisionChanges, cds.watchBufferLength)
 	errs := make(chan error, 1)
 
+	features, err := cds.Features(ctx)
+	if err != nil {
+		errs <- err
+		return updates, errs
+	}
+
+	if !features.Watch.Enabled {
+		errs <- datastore.NewWatchUnsupportedErr(fmt.Sprintf("%s. See https://docs.authzed.com/spicedb/enabling-watch-api#crdb", features.Watch.Reason))
```

Comment:
We might change this link

Reply:
I thought of that; I do wonder if we should add some sort of short-link here instead, in case we need to change to where it points